rq/worker.py

```diff
@@ -160,11 +161,22 @@ def __init__(self, queues, name=None,
 
     def validate_queues(self):
         """Sanity check for the given queues."""
-        if not iterable(self.queues):
-            raise ValueError('Argument queues not iterable.')
         for queue in self.queues:
             if not isinstance(queue, self.queue_class):
-                raise NoQueueError('Give each worker at least one Queue.')
+                raise NoQueueError('Give each worker at least one Queue.\
+                        {} was passed'.format(queue))
+
+    def process_queue_args(self, queue_args):
+        """ allow for a variety arguements to queues """
+        if isinstance(queue_args, six.text_type):
+            return self.queue_class(name = queue_args)
+        else:
+            queue_args = [queue_args] if isinstance(queue_args, six.text_type) else queue_args
+            queue_args = [[queue_arg] if isinstance(queue_arg, six.text_type) 
+                    else list(queue_arg) for queue_arg in queue_args]
+            queue_args = itertools.chain.from_iterable(queue_args)
+            return  [self.queue_class(name = queue_arg) if isinstance(queue_arg,six.text_type) 
```

Comment:
Nitpicks: could you please remove the spaces between the `=` in `self.queue_class(name = queue_arg)`? Also, put a space after comma in `isinstance(queue_arg,six.text_type)`